wkcuber/convert_nifti.py

```diff
@@ -0,0 +1,177 @@
+import time
+import logging
+import wkw
+import numpy as np
+from argparse import ArgumentParser
+import os
+import nibabel as nib
+
+from .utils import (
+    add_verbose_flag,
+    open_wkw,
+    WkwDatasetInfo,
+    ensure_wkw,
+    add_distribution_flags,
+    setup_logging,
+)
+
+from .metadata import write_webknossos_metadata
+
+
+def create_parser():
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "source_path", help="Directory containing the source NIFTI dataset."
+    )
+
+    parser.add_argument(
+        "target_path", help="Output directory for the generated WKW dataset."
+    )
+
+    parser.add_argument(
+        "--layer_name",
+        "-l",
+        help="Name of the cubed layer (color or segmentation).",
+        default="color",
+    )
+
+    parser.add_argument(
+        "--dtype",
+        "-d",
+        help="Target datatype (e.g. uint8, uint16, uint32).",
+        default="uint8",
+    )
+
+    parser.add_argument(
+        "--convert_folder",
```

Comment:
I'd remove this argument and change the description of `source_path` to `Path to NIFTY file or to a directory if multiple NIFTI files should be converted. In the latter case, also see --color_file and --segmentation_file.`

In the code, you simply check whether the path points to a file or a dir.